src/lib/utils/getEndpoint.ts

```diff
@@ -0,0 +1,33 @@
+export type BackendDestination = 'PORTAL_ENDPOINT' | 'SWC_ENDPOINT'
+// These variables indicate which endpoint should be called
+export const REPO_ENDPOINT: BackendDestination = 'PORTAL_ENDPOINT'
```

Comment:
naming is a little confusing REPO_ENDPOINT = 'PORTAL_ENDPOINT' and PORTAL_ENDPOINT: BackendDestination = 'SWC_ENDPOINT' ?

Reply:
whoops good catch, I will fix that..